code/modules/recycling/conveyor.dm

```diff
@@ -137,61 +175,86 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	var/list/items = locturf.contents - src
 	if(!LAZYLEN(items))//Dont do anything at all if theres nothing there but the conveyor
 		return
+
```

Comment:
why did you add so many spaces into places you didn't touch the code

Reply:
Because it makes it easier to read, the less dense the code is, the easier it is to understand the logic of it, it's less mental workload, etc.